File: baekjoon_all/10000+/boj_10870.py
# Solve 2022-05-25
# Update 2023-03-03

n = int(input())

dp = [0] * 21
dp[1] = 1
for i in range(2, n + 1):
    dp[i] = dp[i - 2] + dp[i - 1]

print(dp[n])

File: baekjoon_all/10000+/boj_10870.cpp
// Solve 2023-03-03
// Update 2023-12-20

#include <bits/stdc++.h>
using namespace std;

#define FASTIO ios_base::sync_with_stdio(false);cin.tie(NULL); // boj_15552.cpp
#define SETPRECISION(n) cout << fixed;cout.precision(n); // boj_1008.cpp
#define SIZE(v) (int)v.size()
#define ALL(v) v.begin(),v.end()
using ll = long long;

int main() {
    FASTIO;

    int n;
    cin >> n;

    int dp[21] = { 0, 1 };

    for (int i = 2; i <= n; i++) {
        dp[i] = dp[i - 2] + dp[i - 1];
    }

    cout << dp[n] << '\n';

    return 0;
}
